cpp/src/slice2swift/SwiftUtil.h

```diff
@@ -8,94 +8,78 @@
-
-        void writeProxyDocSummary(IceInternal::Output&, const InterfaceDefPtr&, const std::string&);
-        void writeServantDocSummary(IceInternal::Output&, const InterfaceDefPtr&, const std::string&);
-
-        std::string paramLabel(const std::string&, const ParameterList&);
-        std::string operationReturnType(const OperationPtr&);
-        std::string operationReturnDeclaration(const OperationPtr&);
-
-        std::string typeToString(const TypePtr&, const ContainedPtr&, bool = false);
-
-        std::string getUnqualified(const std::string&, const std::string&);
-        std::string modeToString(Operation::Mode);
-        std::string getOptionalFormat(const TypePtr&);
-
-        static bool isNullableType(const TypePtr&);
-
-        /// Returns a string representing the Swift type `contained` maps to.
-        ///
-        /// If `currentModule` is _not_ provided, the type-string is fully qualified.
-        /// Otherwise, the type-string is qualified relative to `currentModule`.
-        std::string getRelativeTypeString(const ContainedPtr& contained, const std::string& currentModule = "");
-
-        std::string getValue(const std::string&, const TypePtr&);
-        void writeConstantValue(
-            IceInternal::Output& out,
-            const TypePtr&,
-            const SyntaxTreeBasePtr&,
-            const std::string&,
-            const std::string&,
-            bool optional = false);
-        void writeDefaultInitializer(IceInternal::Output&, bool, bool);
-        void writeMemberwiseInitializer(IceInternal::Output&, const DataMemberList&, const ContainedPtr&);
-        void writeMemberwiseInitializer(
-            IceInternal::Output&,
-            const DataMemberList&,
-            const DataMemberList&,
-            const DataMemberList&,
-            const ContainedPtr&,
-            bool rootClass);
-        void writeMembers(IceInternal::Output&, const DataMemberList&, const ContainedPtr&, int = 0);
-
-        void writeMarshalUnmarshalCode(
-            ::IceInternal::Output&,
-            const TypePtr&,
-            const ContainedPtr&,
-            const std::string&,
-            bool,
-            std::int32_t = -1);
-
-        bool usesMarshalHelper(const TypePtr&);
-        void writeMarshalInParams(::IceInternal::Output&, const OperationPtr&);
-        void writeMarshalAsyncOutParams(::IceInternal::Output&, const OperationPtr&);
-        void writeUnmarshalInParams(::IceInternal::Output&, const OperationPtr&);
-        void writeUnmarshalOutParams(::IceInternal::Output&, const OperationPtr&);
-        void writeUnmarshalUserException(::IceInternal::Output& out, const OperationPtr&);
-        void writeSwiftAttributes(::IceInternal::Output&, const MetadataList&);
-        void writeProxyOperation(::IceInternal::Output&, const OperationPtr&);
-        void writeDispatchOperation(::IceInternal::Output&, const OperationPtr&);
```

Comment:
Well, except for these 2 functions, which were inlined and deleted from here.
Other than them, there were no other changes here.